tests/src/test/scala/com/nvidia/spark/rapids/BatchWithPartitionDataSuite.scala

```diff
@@ -55,48 +55,46 @@ class BatchWithPartitionDataSuite extends RmmSparkRetrySuiteBase with SparkQuery
     // This test uses single-row partition values that should throw a GpuSplitAndRetryOOM exception
     // when a retry is forced.
     val maxGpuColumnSizeBytes = 1000L
-    withGpuSparkSession(_ => {
```

Comment:
why remove this?

Reply:
Is this because of the RMM is initialized twice thing?

Reply:
yes exactly. It was only needed because this test wanted SparkSession to be defined, such that other code in the init didn't NPE (it didn't need Spark at all otherwise).